building/build-debs/homeworld-keysystem/src/keyserver/token/scoped/token.go

```diff
@@ -37,7 +38,10 @@ func generateTokenID() string {
 	if err != nil {
 		panic(err)
 	}
-	return base64.RawStdEncoding.EncodeToString(out)
+
+	hash := base64.RawStdEncoding.EncodeToString(out)
+	hashSha256 := sha256.Sum256([]byte(hash))
+	return hash + base64.RawStdEncoding.EncodeToString(hashSha256[:])[0:2]
```

Comment:
I suggest adding a test to ensure that this function always generates valid tokens

Reply:
Done